Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM python:buster
+
+RUN apt-get update
+RUN apt-get -y install curl git rsync \
+    # Node version 14
+    && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+    && apt-get install nodejs
+
+
+RUN git clone https://github.com/codetheweb/MTU-Transfer-Course-Gatherer.git /app/
+WORKDIR /app/
+
+# Install Python dependencies
+
+RUN pip3 install --upgrade pip -r requirements.txt
+RUN python3 ./manage.py makemigrations
+RUN python3 ./manage.py migrate
+RUN python3 ./banwebScrape.py
```

Comment:
In a raw docker environment, I would agree. However more the point of this was a "Get me an environment that will work in one shot". Rather than:
Build the image
Deploy the image
Setup your own custom docker execs that go into the image and modify the data state.

Reply:
I gotcha.  As long as something like `docker exec courses python3 ./banwebScrape.py && python3 ./manage.py custom_db_sync /tmp/writer.csv` works then that's fine to have seed data in the image.

Reply:
https://github.com/Kiro47/MTU-Transfer-Course-Gatherer#run-the-scrape-script-to-collect-the-up-to-date-course-list